com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/BloomPrefilter.compute

```diff
@@ -37,11 +55,11 @@ void KMain(uint3 dispatchThreadId : SV_DispatchThreadID)
         float2(-0.4, -0.9)
     };
 
-    float3 color0 = SAMPLE_TEXTURE2D_X_LOD(_InputTexture, sampler_LinearClamp, ClampAndScaleUVForBilinear(_TexelSize.zw * kTaps[0] + uv, _TexelSize.zw), 0.0).xyz;
-    float3 color1 = SAMPLE_TEXTURE2D_X_LOD(_InputTexture, sampler_LinearClamp, ClampAndScaleUVForBilinear(_TexelSize.zw * kTaps[1] + uv, _TexelSize.zw), 0.0).xyz;
-    float3 color2 = SAMPLE_TEXTURE2D_X_LOD(_InputTexture, sampler_LinearClamp, ClampAndScaleUVForBilinear(_TexelSize.zw * kTaps[2] + uv, _TexelSize.zw), 0.0).xyz;
-    float3 color3 = SAMPLE_TEXTURE2D_X_LOD(_InputTexture, sampler_LinearClamp, ClampAndScaleUVForBilinear(_TexelSize.zw * kTaps[3] + uv, _TexelSize.zw), 0.0).xyz;
-    float3 color4 = SAMPLE_TEXTURE2D_X_LOD(_InputTexture, sampler_LinearClamp, ClampAndScaleUVForBilinear(_TexelSize.zw * kTaps[4] + uv, _TexelSize.zw), 0.0).xyz;
+    float3 color0 = SAMPLE_TEXTURE2D_X_LOD(_InputTexture, sampler_LinearClamp, ClampAndScaleUVForBilinear(_ScreenSize.zw * kTaps[0] + uv, _ScreenSize.zw), 0.0).xyz;
```

Comment:
Have you tested this change by rescaling the window? 

The ScreenSize is the actual widht/height (so with the RTHandleScale baked in), but ClampAndScaleUVForBilinear scale with that RTHandleScale, so I am a bit unsure about this. 

Also, why the change at all? _TexelSize.zw is the size of the first mip.

Reply:
**IF** this is what you wanted to do 

Then you should use 

ClampAndScaleUVForBilinear(float2 UV) 

as it embeds _ScreenSize.zw as texel size already.

Reply:
That actually seems correct , but thanks for the check, I didn't do it. So I updated the function call to use the overload